packages/template-retail-react-app/app/pages/social-login-redirect/index.jsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React, {Fragment} from 'react'
+import {FormattedMessage} from 'react-intl'
+import {Box, Container, Stack, Text} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {BrandLogo} from '@salesforce/retail-react-app/app/components/icons'
+
+const SocialLoginRedirect = () => {
```

Comment:
Is it intended that we don't have this new page wired up in the router for the application? Is that because the URL is configurable?

It would be nice if we reduce the setup steps for this feature and not have to as them to wire up the router. We can avoid some documentation if we do that.

Reply:
Forgive my ignorance, but how is this page automatically navigated away from? I'm guessing that is happening in the _app-config component or _app? 🤔 

Is there a reason that I'm missing as to why we aren't co-locating that code?

Reply:
Oops - I initially added this page in `routes.jsx` but it seems that the changes weren't pushed. I have included these changes now.

And for the automatic redirect, it will be similar to how the login flow works. In the login page [here](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/template-retail-react-app/app/pages/login/index.jsx#L79-L88), if customer `isRegistered`, then it redirects. This code will be located here in this file. I was intending this PR to just implement the UI but I can also add it now.